sdk/epoch-rewards/src/lib.rs

```diff
@@ -2,15 +2,27 @@
 //!
 //! [sv]: https://docs.solanalabs.com/runtime/sysvars#epochrewards
 //!
-//! The sysvar ID is declared in [`sysvar::epoch_rewards`].
+//! The sysvar ID is declared in [`solana_program::sysvar::epoch_rewards`].
 //!
-//! [`sysvar::epoch_rewards`]: crate::sysvar::epoch_rewards
+//! [`solana_program::sysvar::epoch_rewards`]: https://docs.rs/solana-program/latest/solana_program/sysvar/epoch_rewards/index.html
```

Comment:
And here `solana_epoch_rewards` with the updated docs link?

Reply:
We can use the local documentation when it's within the same crate thankfully, so I went with that